website/docs/dbt-cli/install/docker.md

```diff
@@ -3,8 +3,117 @@ title: "Use Docker to install dbt"
 description: "You can use Docker to install dbt and adapter plugins from the command line."
 ---
 
-By v1.0.0, dbt Core and all adapter plugins maintained by dbt Labs will be available as official Docker images,
-and available from a public registry.
-We recommend you use Docker to install in production because it includes dbt Core and all of its dependencies. You might might also use a Docker to install and develop locally if you don't have your python environment set up.
+You will find dbt Core and all adapter plugins maintained by dbt Labs available as [Docker](https://docs.docker.com/) images. We recommend you use Docker to install dbt in production because it includes dbt Core, one or more database adapters, and all dependencies. You can use Docker to install and develop locally if you don't have a python environment set up.
 
-More information coming soon!
+### Install a dbt Docker image from Github Packages
+
+Official dbt Docker images are hosted on our [GitHub packages page](https://github.com/orgs/dbt-labs/packages?visibility=public).  We maintain images and tags for each database adapter as well as a `latest` and `<major version>.<minor version>.latest` tag.  They can be installed by utilizing the `docker pull` command:
```

Comment:
It seems like `latest` can be dropped, right?
```suggestion
Official dbt Docker images are hosted on our [GitHub packages page](https://github.com/orgs/dbt-labs/packages?visibility=public).  We maintain images and tags for each database adapter as well as a `latest` and `<major version>.<minor version>` tag.  They can be installed by utilizing the `docker pull` command:
```

Reply:
@runleonarun We maintain two types of "latest" tags:
- Absolute `latest` (newest stable release)
- Major.Minor-specific `latest`, e.g. `1.0.latest`

As a concrete example: After we release v1.1.0, the `latest` tag will start installing v1.1. For users who are still on v1.0, the `1.0.latest` tag will still be available, and it will install the latest compatible patch (v1.0.x).

Some of this will be clearer once we have clearer version policies: #1061

Reply:
@iknox-fa I do think we should clarify here when each `latest` tag is updated, especially for plugins. That's a question from a community member right now, since our plugin `latest` images haven't yet updated to install `dbt-core==1.0.2`: https://github.com/dbt-labs/dbt-core/issues/4756

Reply:
@jerco I assume you mean we should reference our current release schedule here?

Reply:
@runleonarun 
> It seems like latest can be dropped, right?
Nope, as @jerco points out.

Reply:
@iknox-fa Yeah! We've got a page now that we could link to: https://docs.getdbt.com/docs/core-versions

I think the question was: Are we rebuilding plugin images (e.g. `dbt-bigquery`) when new patch versions of `dbt-core` become available, so that they get access to the latest fixes from `dbt-core`? If so, do we update either the version tags or `latest` tags? If I understand right, we aren't doing this, since [`dbt-bigquery@1.0.latest`](https://github.com/dbt-labs/dbt-bigquery/pkgs/container/dbt-bigquery/17112343?tag=1.0.latest) hasn't been updated for a few months